src/libraries/System.Runtime.InteropServices/tests/TestAssets/SharedTypes/ComInterfaces/IEmpty.cs

```diff
@@ -5,14 +5,12 @@
 using System.Runtime.InteropServices;
 using System.Runtime.InteropServices.Marshalling;
 
-namespace ComInterfaceGenerator.Tests
+namespace SharedTypes.ComInterfaces
 {
     [GeneratedComInterface]
-    [InterfaceType(ComInterfaceType.InterfaceIsIUnknown)]
-    [Guid("2c3f9903-b586-46b1-881b-adfce9af47b1")]
-    public partial interface IComInterface1
+    [Guid(_guid)]
+    internal partial interface Empty
```

Comment:
I noticed in your generated code PR that the `CreateManagedVirtualFunctionTable` for this interface is returning `null`. It should return a pointer to a vtable with only the `IUnknown` methods.

Reply:
Good catch, I've updated here and in the generated code PR.